extension/src/openvic-extension/singletons/GameSingleton.cpp

```diff
@@ -387,8 +378,8 @@ void GameSingleton::set_viewed_country_by_province_index(int32_t province_index)
 	InstanceManager* instance_manager = get_instance_manager();
 	ERR_FAIL_NULL(instance_manager);
 
-	ProvinceInstance const* province_instance =
-		instance_manager->get_map_instance().get_province_instance_by_index(province_index);
+	ProvinceInstance const* province_instance = instance_manager->get_map_instance().get_province_instance_by_index(province_index
+	);
```

Comment:
The `);` on its own at the end of the line here is very weird, breaking after the `=` or at both ends of the `()` would look better IMO